src/store/epics/chatEpics.js

```diff
@@ -28,8 +28,13 @@ import {
 import ChatHandler from './helpers/ChatHandler';
 import { combineEpics } from 'redux-observable';
 import createDetectVisibilityObservable from './helpers/createDetectPageVisibilityObservable';
+import { Capacitor } from '@capacitor/core';
+
+let wsUrl = `${window.location.origin.replace('http', 'ws')}/api/ws/messages/all`;
+if (Capacitor.isNativePlatform()) {
```

Comment:
I did what was quickest to get something working, but we should probably do this conditional once, in one place, and define what are effectively global constants which we then export, then import in places like this.

But... IT WORKS!!!